dpppt-backend-sdk/dpppt-backend-sdk-interops/README.md

```diff
@@ -0,0 +1,153 @@
+# Interoperability
+
+## Introduction
+
+The Interoperability Service operates as a single standalone micro service. It periodically connects to the Federation Gateway Service to exchange any new keys from and to the `dpppt-backend-sdk-ws`-Database. The service uses mTLS to connect to the Federation Gateway and a default database connector to connect to the database. 
+
+Hence, the service needs the capability to connect to external services. Further, in order for mTLS to work, the connection _MUST_ either _NOT_ be proxied (peer to peer connection), or the load balancer used _MUST_ support establishing mTLS connection based on certificate finger/thumb prints.
+
+Further, as the `dpppt-backend-sdk-ws` the Interoperability Service needs access to the database storing the EN-Keys.
+## Federation Gateway Options
+As for the `dpppt-backend-sdk-ws` different configuration classes provide the option to parameterize the runtime behavior of the interoperability service. 
+
+Most of the properties found in the `WSBaseConfig` are similar to the ones found in the `WSBaseConfig` of the `dpppt-backend-sdk-ws`. Hence we only go into details for interoperability specific options. In contrast to the `dpppt-backend-sdk-ws` the interoperability service also uses configuration beans (c.f. `HubConfigs` and `EfgsGatewayConfig`).
+
+### WSBaseConfig
+- `interops.origin.country`: This property holds the [ISO-3166-1-Alpha-2](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) identifier for the country in which the service runs. This property will be added to keys which were uploaded via the `dpppt-backend-sdk-ws` and hence did not come from the federation gateway. *DEFAULT: null* - since this property needs to be specified by the country starting the service, it should fail if it was forgotten (not specified).
+
+- `interops.efgs.dsosfilter.enabled`: Flag indicating if the `days since onset of symptoms filters` (DsosFilter) should be enabled or not. See the section about `Days Since Onset of Symptoms` for further information on the encoding and purpose of `dsos`. If enabled the filter will be injected into the filter pipeline and keys not matching the following criteria are filtered out. *DEFAULT: true*
+
+- `interops.efgs.dsosfilter.symptomaticOnsetKnown.dropDaysBeforeOnset`: Drop keys which are older than `onset - n` and the onset of symptoms is known. Negative values reference to days *before* onset. *DEFAULT: -2*
```

Comment:
dropDaysBeforeOnset implies a positive number. `onset - n` also suggests `n` is `dropDaysBeforeOnset` so also a positive number. it's quite confusing that the configured property has to be negative. same goes for the other dsosfilter drop days properties

Reply:
We should document it like this for know and open an issue to tackle the confusing property. @UBaggeler  or what is your opinion?